plugins/src/webrtcsink/imp.rs

```diff
@@ -407,6 +418,24 @@ fn configure_encoder(enc: &gst::Element, start_bitrate: u32) {
                 enc.set_property("keyframe-period", 2560u32);
                 enc.set_property_from_str("rate-control", "cbr");
             }
+            "nvv4l2h264enc" => {
+                enc.set_property("bitrate", 4096000u32 / 1000);
+                enc.set_property("insert-sps-pps", true);
+                enc.set_property_from_str("preset-level", "UltraFastPreset");
+                enc.set_property("maxperf-enable", true);
+                enc.set_property("insert-vui", true);
+                enc.set_property("idrinterval", 256u32);
+                enc.set_property("insert-sps-pps", true);
+                enc.set_property("insert-aud", true);
+                enc.set_property_from_str("control-rate", "variable_bitrate");
```

Comment:
constant bitrate is much preferable for congestion control purposes, please change to that :)